.github/ISSUE_TEMPLATE/feature-request.md

```
@@ -20,3 +21,8 @@ A clear and concise description of approach to be followed.
 
 **Additional context**
 Add any other context or screenshots about the feature request here.
+
+**Potential internship candidates**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)